holoviews/core/options.py

```diff
@@ -1106,6 +1114,25 @@ def lookup(cls, backend, obj):
 
 
     @classmethod
+    def transfer_options(cls, obj, new_obj, drop=[]):
+        """
+        Transfers options for all backends from one object to another.
+        Drops any options defined in the supplied drop list.
+        """
```

Comment:
Seems useful though it definitely needs some examples (even if only in unit tests).